Source/WebKit/UIProcess/API/Cocoa/_WKWebExtensionBookmarks.h

```diff
@@ -0,0 +1,79 @@
+// Copyright © 2025  All rights reserved.
+
+#import <Foundation/Foundation.h>
+
+
+@class WKWebExtensionContext;
+@protocol _WKWebExtensionBookmark;
+
+WK_HEADER_AUDIT_BEGIN(nullability, sendability)
+
+/*!
+ @abstract Constants used by ``_WKWebExtensionBookmark`` to indicate the type of a bookmark node.
+ @constant _WKWebExtensionBookmarkTypeBookmark  Indicates the node is a bookmark with a URL.
+ @constant _WKWebExtensionBookmarkTypeFolder  Indicates the node is a folder that can contain other bookmarks or folders.
+ @constant _WKWebExtensionBookmarkTypeSeparator  Indicates the node is a separator.
+ */
+typedef NS_ENUM(NSInteger, _WKWebExtensionBookmarkType) {
+    _WKWebExtensionBookmarkTypeBookmark,
+    _WKWebExtensionBookmarkTypeFolder,
+} NS_SWIFT_NAME(_WKWebExtension.BookmarkType) WK_API_AVAILABLE(macos(15.4), ios(18.4), visionos(2.4));
```

Comment:
Nit: I think we should use the `TBA` macros instead of putting an actual version number in here.

1) We don't know when this will ship
2) I think this is already incorrect with the iOS 26 version number

Reply:
Ditto this for all of the other parts of the delegate.